### PR TITLE
2023 November Formal Submissions

### PR DESCRIPTION
**Pull request by blimpa**
- Implemented the revised template
- Changed status from RFC to Formal Submission (some authors have not confirmed yet but most likely will)
- Updated with latest changes on the forum

### CODE REVIEW

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP17.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ MIP102c2-SP#: 17
 MIP to be amended: MIP104, MIP106, MIP107, MIP108, MIP113
 Author(s): @IamMeeoh
 Contributors: @vigilant, @Bonapublica, @Palc
+General Edit or Article 1 Edit: Article 1 Edit
```
Comment (0xldr) [inline MIP102/MIP102c2-Subproposals/MIP102c2-SP17.md:10]: We are not sure that this information will work as is presumably intended. The MIPs Portal does not appear to parse it when used elsewhere such as in https://mips.makerdao.com/mips/details/MIP102c2SP18#specification

If this information is desired to be displayed, we suggest a tag might be a better method until the portal UI can reflect adding this information as a separate element of the preamble

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP18.md [hunk 2]
```diff
@@ -34,3 +34,18 @@ This subproposal will amend MIP104 to provide clarifying edits to [recently upda
 ### Amendment Pull Request
 
 https://github.com/makerdao/mips/pull/975
+
```
Comment (0xldr) [inline MIP102/MIP102c2-Subproposals/MIP102c2-SP18.md:37]: please update the forum link in the preamble to https://forum.makerdao.com/t/mip102c-sp18-mip-amendment-subproposals/22027

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP19.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ MIP102c2-SP#: 19
 MIP to be amended: MIP113
 Author(s): @PALC
 Contributors: Sovereign Finance AVC, Bonapublica
+General Edit or Article 1 Edit: General Edit
```
Comment (0xldr) [inline MIP102/MIP102c2-Subproposals/MIP102c2-SP19.md:10]: See our previous comment

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP19.md [hunk 4]
```diff
@@ -16,14 +17,15 @@ Forum URL: https://forum.makerdao.com/t/mip102c2-sp18-mip-amendment-subproposal/
 
 ## Specification
```
Comment (0xldr) [inline MIP102/MIP102c2-Subproposals/MIP102c2-SP19.md:18]: Please update the forum URL to https://forum.makerdao.com/t/mip102c2-sp18-mip-amendment-subproposal/22335

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP20.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ Author(s): @Rune
 Contributors:
 General Edit or Article 1 Edit: General Edit
 Tags: 
-Status: RFC
+Status: Formal Submission
 Date Proposed: 2023-10-05
 Date Ratified: <yyyy-mm-dd>
 Forum URL: https://forum.makerdao.com/t/mip102c2-spxx-mip-amendment-subproposal/22334
```
Comment (0xldr) [inline MIP102/MIP102c2-Subproposals/MIP102c2-SP20.md]: Update forum URL to: https://forum.makerdao.com/t/mip102c2-sp20-mip-amendment-subproposal/22334

**Review by 0xldr — CHANGES_REQUESTED** [review]
We have conducted a review of the changes proposed in this Pull Request. Please note the relevant comments.

We note that MIP102c2-SP19 has not been formally submitted by the authors as yet, and as such remains in RFC, despite forming part of the formal submissions PR. This is not a problem but we are noting it here in case any future readers are confused.

Outside of the noted updates to forum URLs, we have not reviewed the underlying files for accuracy but have only reviewed the changes being proposed in this Pull Request. We assume that appropriate due diligence was done at the time these files were initially submitted.

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP17.md (hunk 1 above)
Comment (blimpa) [reply MIP102/MIP102c2-Subproposals/MIP102c2-SP17.md:10]: That is correct. The Portal doesn't know what to do with these at this point in time. This field is supposed (so is my understanding) to make extra-sure authors are aware that they need to choose between these two alternatives. Tags are a good idea, though. I can do a mass tagging of all MIP102c2 subproposals later (without delaying the merging of this PR)

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP18.md (hunk 2 above)
Comment (blimpa) [reply MIP102/MIP102c2-Subproposals/MIP102c2-SP18.md:37]: (That's the same URL)

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP17.md (hunk 1 above)
Comment (blimpa) [reply MIP102/MIP102c2-Subproposals/MIP102c2-SP17.md:10]: On second thought, I'll add the tags for these now

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP18.md (hunk 2 above)
Comment (blimpa) [reply MIP102/MIP102c2-Subproposals/MIP102c2-SP18.md:37]: Updated. (The previous URL pointed to the same place [it's resolved by Discourse], but I agree that this one is nicer :) )

**Review by 0xldr — APPROVED** [review]
Approved

**Pull request merged**